### PR TITLE
Fix failure alert tests

### PR DESCRIPTION
**Pull request by elias-ba**
## Related issue

Fixes #

## Review checklist

- [ ] I have performed a **self-review** of my code
- [ ] I have verified that all appropriate **authorization policies** have been implemented and tested
- [ ] If needed, I have updated the **changelog**
- [ ] Product has **QA'd** this feature

### CODE REVIEW

File: test/lightning/failure_alert_test.exs [hunk 1]
```diff
@@ -176,77 +175,77 @@ defmodule Lightning.FailureAlertTest do
...
+
+      push(socket, "run:complete", %{
+        "run_id" => run.id,
+        "output_dataclip_id" => Ecto.UUID.generate(),
+        "output_dataclip" => ~s({"foo": "bar"}),
+        "reason" => "normal"
+      })
+
+      # Wait a bit to make sure run:complete is handled
+      # TODO: Check with Taylor DOWNS and team if this is good enough
```
Comment (elias-ba) [inline test/lightning/failure_alert_test.exs:245]: @taylordowns2000 please in the AM, would you mind checking this out ?

Comment (taylordowns2000) [reply test/lightning/failure_alert_test.exs:245]: looks good. reason "normal" should be reason "failed", but otherwise it's great.

**Review by taylordowns2000 — APPROVED** [review]
@elias-ba , this looks great. i'm making a small change (reason normal to reason fail) but otherwise it seems to test the right stuff well!

**Review by taylordowns2000 — CHANGES_REQUESTED** [review]
ahhh sorry @elias-ba ... i was wrong. it looks like this alerter is now bound to `run:complete` and it _should_ be bound to `attempt:complete`!

can you shift this around so that it notifies the user if an **attempt** fails, not a run?

**Comment by elias-ba** [comment]
> ahhh sorry @elias-ba ... i was wrong. it looks like this alerter is now bound to `run:complete` and it _should_ be bound to `attempt:complete`!
> 
> can you shift this around so that it notifies the user if an **attempt** fails, not a run?

@taylordowns2000 this is now handled. Please review again and let me know

**Review by taylordowns2000 — APPROVED** [review]

**Pull request merged**